CodeSniffer/Standards/PSR1/Sniffs/Methods/CamelCapsMethodNameSniff.php

```diff
@@ -98,4 +95,24 @@ protected function processTokenOutsideScope(PHP_CodeSniffer_File $phpcsFile, $st
     }//end processTokenOutsideScope()
 
 
+    /**
+     * Returns true if the specified method name refers to a magic method.
+     *
+     * @param string $methodName The name of the method that should be checked.
+     *
+     * @return bool
+     */
+    private function _isMagicMethod($methodName)
```

Comment:
Would this be worth pulling up into `Generic_Sniffs_NamingConventions_CamelCapsFunctionNameSniff`?